components/molecule/modal/src/index.js

```diff
@@ -0,0 +1,186 @@
+import PropTypes from 'prop-types'
+import React, {Component} from 'react'
+import cx from 'classnames'
+import {SUPPORTED_KEYS} from './config'
+import {suitClass} from './helpers'
+
+const toggleWindowScroll = disableScroll => {
+  window.document.body.classList.toggle('is-MoleculeModal-open', disableScroll)
+}
+
+class MoleculeModal extends Component {
+  _contentRef = React.createRef()
+  _wrapperRef = React.createRef()
+
+  componentDidMount() {
+    document.addEventListener('keydown', this._onKeyDown)
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this._onKeyDown)
```

Comment:
we need to add `this._toggleWindowScroll(false)`

Reply:
done!